config/environment.js

```diff
@@ -11,7 +11,13 @@ module.exports = function(environment) {
     EmberENV: {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
-        // e.g. 'with-controller': true
+        // e.g. EMBER_NATIVE_DECORATOR_SUPPORT: true
+        EMBER_NATIVE_DECORATOR_SUPPORT: true,
```

Comment:
These feature flags are not enabled, I don't think because we are no on a canary build. Is that the intention?